src/prettier-eslint.js

```diff
@@ -0,0 +1,8 @@
+try {
+  // if `prettier-eslint` is installed by the user manually
+  module.exports = require('prettier-eslint');
+} catch (err) {
+  /* istanbul ignore next */
+  // it is an internal dependency using `prettier-eslint` as fallback
+  module.exports = require('@prettier/eslint');
```

Comment:
`prettier-eslint` is always user defined.

Reply:
good enough for me!

Reply:
If we're able to log here we could even let users know our behavior explicitly depending on which path occurs.

Reply:
Sure, any idea how should we log?

Reply:
It looks like we use something called `messageformat` (https://github.com/prettier/prettier-eslint-cli/blob/master/src/messages.js) AND `loglevel-colored-level-prefix` for various logging. I think `loglevel-colored-level-prefix` looks more appropriate, so I'd suggest using that dependency.

```js
import getLogger from https://github.com/kentcdodds/loglevel-colored-level-prefix;

const logger = getLogger({ prefix: 'prettier-eslint-cli' });

try {
  // if `prettier-eslint` is installed by the user manually
  module.exports = require('prettier-eslint');
  logger.info('We detected that you installed a version of `prettier-eslint` and are using that as if it were a peer dependency.');
} catch (err) {
  /* istanbul ignore next */
  // it is an internal dependency using `prettier-eslint` as fallback
  module.exports = require('@prettier/eslint');
}
```

** here is somehow the version number

Reply:
something like that! tweak what you want 🙏🏼

Reply:
I don't know whether should we log at `try` or `catch` instead. `try` means it's peer dependency, `catch` means it's the fallback one. Maybe we should notice the user `fallback` instead?

Reply:
Do both 🤷🏼

Reply:
> Do both 🤷🏼

😅 Too verbose